packages/@react-aria/table/docs/useTable.mdx

```diff
@@ -0,0 +1,723 @@
+
+## Features
+
+A table can be built using the [&lt;table&gt;](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table), [&lt;thead&gt;](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/thead),
+[&lt;tbody&gt;](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tbody), and other table specific HTML elements, but is very limited in functionality especially when it comes to user interactions.
+`useTable` helps achieve accessible and interactive table components that can be styled as needed.
+
+* Exposed to assistive technology as a `grid` using ARIA
+* Support for single, multiple, or no row selection
+* Support for disabled rows
+* Labeling support for accessibility
+* Support for mouse, touch, and keyboard interactions
+* Support for keyboard navigation between columns, rows, cells, and in-cell focusable items
+* Column sorting
+* Multiple [row header](https://www.w3.org/TR/wai-aria-1.1/#rowheader) support
+* Automatic scrolling support during keyboard navigation
+* Typeahead to allow focusing rows by typing text
+* Virtualized scrolling support for performance with large tables
+
+## Anatomy
+
+<Anatomy />
+
+A table consists of a container element, with columns and rows of cells containing data inside. The cells within a table may contain focusable elements or plain text content.
+If the table supports row selection, each row includes a selection checkbox. Additionally, a "select all" checkbox is displayed as the first column header if the table supports multiple row selection.
+
+`useTable`, `useTableCell`, `useTableColumnHeader`, `useTableRow`, and `useTableRowHeader` handle keyboard, mouse, and other interactions to support
+row selection, in table navigation, and overall focus behavior. Those hooks, along with `useTableRowGroup`, also handle exposing the table and its contents
+to assistive technology using ARIA. `useTableSelectAllCheckbox` and `useTableSelectionCheckbox` handle row selection and associating each checkbox with its respective rows
+for assistive technology.
+
+`useTable` returns props that you should spread onto the table container element:
+
+<TypeContext.Provider value={docs.links}>
+  <InterfaceType properties={docs.links[docs.exports.useTable.return.id].properties} />
+</TypeContext.Provider>
+
+`useTableColumnHeader` returns props for an individual table column header:
+
+<TypeContext.Provider value={docs.links}>
+  <InterfaceType properties={docs.links[docs.exports.useTableColumnHeader.return.id].properties} />
+</TypeContext.Provider>
+
+`useTableRow` returns props for an individual table row:
+
+<TypeContext.Provider value={docs.links}>
+  <InterfaceType properties={docs.links[docs.exports.useTableRow.return.id].properties} />
+</TypeContext.Provider>
+
+`useTableRowGroup` returns props for a element containing one or more rows of column headers or cells. This is often equivalent
+to the header, body, and footer of the table:
```

Comment:
We have an example of this for the column header row and the rows in the body, but not for the footer of the table. Decided to mention the footer since users can use this hook for a footer row